server/src/main/java/org/candlepin/model/Consumer.java

```diff
@@ -387,20 +388,48 @@ public String getOwnerId() {
         return ownerId;
     }
 
-    public String setOwnerId(String ownerId) {
-        return ownerId;
+    public Consumer setOwnerId(String ownerId) {
+        this.ownerId = ownerId;
+        return this;
     }
 
     /**
-     * Associates an owner to this Consumer.
-     * @param owner owner to associate to this Consumer.
+     * Fetches the owner of this consumer, if the owner ID is set.
+     * <p></p>
+     * This method performs an on-demand lookup of the owner
+     *
+     * @return
+     *  The owner of this consumer, if the owner ID is populated; null otherwise.
      */
-    public void setOwner(Owner owner) {
-        if (owner != null) {
-            this.ownerId = owner.getId();
-        }
+    @Override
+    @JsonIgnore
+    @Formula("SELECT o FROM Owner WHERE o.id = owner_id")
+    @Basic(fetch = FetchType.LAZY)
+    public Owner getOwner() {
+        return null;
     }
 
+    // /**
```

Comment:
Delete this?  Seems obsolete now.

Reply:
Oh yeah. I need to verify that the formula call works as I expect it to and add tests around it.